dev/src/main/java/org/eclipse/codewind/intellij/ui/constants/UIConstants.java

```diff
@@ -10,11 +10,25 @@
  *******************************************************************************/
 package org.eclipse.codewind.intellij.ui.constants;
 
+import com.intellij.openapi.actionSystem.DataKey;
+
+import com.intellij.openapi.externalSystem.model.ProjectSystemId;
+import com.intellij.openapi.util.Key;
+import org.eclipse.codewind.intellij.core.console.SocketConsole;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
 public class UIConstants {
+
+    @NotNull
+    public static final Map<String, Key<SocketConsole>> LOGS_KEYMAP = new HashMap<>();
```

Comment:
This definition of `LOGS_KEYMAP` and the import statements above shouldn't be part of this change.

Reply:
Hmm yes indeed.